From 0b6f34f8ac863315b04ff0296bd31682c74eee53 Mon Sep 17 00:00:00 2001
From: ljf <ljf@rock-chips.com>
Date: Tue, 16 Dec 2014 19:56:18 +0800
Subject: [PATCH] iep driver:

get rid of kfree buffer which allocated using devm_kzalloc.

Signed-off-by: ljf <ljf@rock-chips.com>
---
 drivers/video/rockchip/iep/iep_drv.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/video/rockchip/iep/iep_drv.c b/drivers/video/rockchip/iep/iep_drv.c
index 5368cc56c0ef..8882ed9d4260 100755
--- a/drivers/video/rockchip/iep/iep_drv.c
+++ b/drivers/video/rockchip/iep/iep_drv.c
@@ -996,8 +996,6 @@ static int iep_drv_probe(struct platform_device *pdev)
 err_ioremap:
     wake_lock_destroy(&data->wake_lock);
 err_clock:
-    kfree(data);
-
     return ret;
 }
 
@@ -1032,7 +1030,6 @@ static int iep_drv_remove(struct platform_device *pdev)
     }
 #endif
 
-    //devm_kfree(data);
     return 0;
 }
 
-- 
2.35.3

